index.html

```diff
@@ -2,16 +2,12 @@
 
 <html>
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+  <meta charset="UTF-8"/>
   <title>Spoon-Knife</title>
-  <LINK href="styles.css" rel="stylesheet" type="text/css">
+  <link href="styles.css" rel="stylesheet" type="text/css">
```

Comment:
I changed it to lowercase because in **XHTML** (even though this isn't **XHTML**) it's not allowed